.github/workflows/validate.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       run: cargo clippy -- -D clippy::style
     - name: Build examples
       working-directory: ./examples/hpc/hello-dla
-      run: cargo build --example dla
+      run: cargo build --example dla --target riscv64imac-unknown-none-elf
```

Comment:
This should be defined already in examples/hpc/.cargo/config.toml but I don't necessarily mind it being duplicated here.

Reply:
Removed this